contents/handbook/product/product-manager-role.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Role of a Product Manager
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+# Role of a Product Manager
+
+The role a Product Manager varies enormously by individual and organization, however there are some nuances about what makes a great Product Manager at PostHog and what you can expect from them.
+
+## Why do we have Product Managers?
+
+Product development at PostHog is engineering led, all new projects and existing features have an engineering owner who is the "informed captain" making the final call on everything that they build. Product Managers ensure these owners have the company, industry and customer context they need to make great decisions whilst challenging them to move faster and think bigger.
+
+## How do they act?
+
+* **Accelerating** - They're always looking for opportunities push the team to ship faster, more impact and high quality 
+* **Empowering** - They empower others to make great decisions and elevate their ambition through context and coaching
+* **Thoughtful** - Deeply consider the implications and motivations of product decisions and tailor their approach to get the most out of every team member
+
+## What do they do?
+
+### Strategy
+
+* **Direction** - Set and align the team / company on the most impactful and exciting direction
+* **Goals & Metrics** - Set and align the team / company on ambitious goals and metrics to validate our progress
+* **New Opportunities** - Find new, impactful and exciting opportunities for the team / company to exploit
+
+### Execution
+
+* **Context** - Gather and summarize customer, company and industry context into clear actionable opinions to inform product decisions
+* **Unblock** - Identify bottlenecks and remove them, make fast and thorough calls on the toughest product decisions
+* **Challenge** - Challenge the team to think bigger, solve the most impactful problems and stretch their own skills
+* **Fill Gaps** - If we don't have a role for something but it needs to be done to ship a great product, do it
+* **Communicate** - Over-communicate, over-transparently
```

Comment:
I see communication as one of the most important execution actions / habits a PM can have. A great communicator will be more effective at all the other execution steps.